model/src/main/proto/exploration.proto

```diff
@@ -322,6 +322,39 @@ message UserAnswer {
   WrittenTranslationContext written_translation_context = 6;
 }
 
+// Corresponds to a raw item selection answer that user has submitted.
+message ItemSelectionRawAnswer {
+  repeated int32 selected_indexes = 1;
+}
+
+// Corresponds to a raw user answer that user has submitted.
+message RawUserAnswer {
+  oneof answer_input_type {
+    // Just indicates the presence of the type.
+    bool continue = 1;
```

Comment:
Is this one actually used? It doesn't seem to be.

Reply:
Removed unused.